### DISCUSSION TITLE
Teaser example gives error: `No usable default <class 'exchangelib.folders.known_folders.Inbox'> folders`

### DISCUSSION THREAD

**Discussion by kostrykin** [ecederstrand/exchangelib#1334] [Q&A] [answered]
When running the Teaser example on my own mail account, I get the following error:

> exchangelib.errors.ErrorFolderNotFound: No usable default <class 'exchangelib.folders.known_folders.Inbox'> folders

I couldn't really find anything useful googling for this error, and since this is my first time trying exchangelib, maybe someone can point me into the direction of what is going wrong here, or what this error means?

**Comment by ecederstrand** [comment, marked as answer]
The error means that we received a list of folders from the server, but couldn't find anything in that list that looks like an Inbox folder, which is what we requested with `account.inbox`. It's possible that your credentials have access to log into the mailbox, but no access to the inbox folder.

To test that, you could enable debug logging to see exactly what exchangelib is requesting and what the server is responding: https://ecederstrand.github.io/exchangelib/#troubleshooting

**Reply by kostrykin** [reply]
Right, the issue was in the credentials! I used the wrong account. Thanks!
